fabfile/config.py

```diff
@@ -12,6 +12,13 @@
 HOME_DIRECTORY = HOME_DIR
 RESULTS_DIRECTORY = os.path.join(HOME_DIR, 'results')
 CITUS_INSTALLATION = os.path.join(HOME_DIR, 'citus-installation')
+RELATIVE_REGRESS_PATH = 'src/test/regress'
+COMMUNITY_REPO = 'community'
```

Comment:
This is not used and we dont need this variable there is already a variable called `CITUS_REPO` we can remove this.

Reply:
Done